libs/portals/shared-modules/delegations/src/lib/navigation.ts

```diff
@@ -35,3 +35,29 @@ export const delegationsNavigation: PortalNavigationItem = {
     },
   ],
 }
+
+export const delegationsNavigationChildrenWithFolder: PortalNavigationItem[] = [
```

Comment:
I was over complicating this because I wanted to avoid creating directory in the navigation in the admin portal... until I rembembered that there is no side navigation there. 

So yes ofc this is just `delegationsNavigationChildren` and used as you suggested 🤦‍♂️